plugins/binary_filter/zlib/CMakeLists.txt

```diff
@@ -34,6 +30,12 @@ function(add_zlib_module)
       FOLDER "Core/Plugins/Compression"
       DEPENDENCIES ${ZLIB_LIBRARIES})
 
+    if(NOT MSVC)
+      target_include_directories(compress_zlib SYSTEM PRIVATE ${ZLIB_INCLUDE_DIRS})
+    endif()
```

Comment:
Do you know why this isn't included on Windows?

Reply:
I think we were linking against the `libz` that comes with Boost (whatever the name is there) as there is no `libz` on Windows. We might be able to fall back to having the user install it (through vcpkg or similar), though.